pages/static/assets/sidebar/components.svg

```diff
@@ -1 +1 @@
-<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M21.625 6.5H2.375A1.377 1.377 0 0 1 1 5.125v-2.75C1 1.617 1.617 1 2.375 1h19.25C22.383 1 23 1.617 23 2.375v2.75c0 .758-.617 1.375-1.375 1.375ZM2.375 1.917a.458.458 0 0 0-.458.458v2.75c0 .253.205.458.458.458h19.25a.458.458 0 0 0 .458-.458v-2.75a.458.458 0 0 0-.458-.458H2.375ZM21.625 14.75H9.71a1.377 1.377 0 0 1-1.375-1.375v-2.75c0-.758.616-1.375 1.375-1.375h11.916c.758 0 1.375.617 1.375 1.375v2.75c0 .758-.617 1.375-1.375 1.375ZM9.71 10.167a.458.458 0 0 0-.459.458v2.75c0 .253.206.458.459.458h11.916a.458.458 0 0 0 .459-.458v-2.75a.458.458 0 0 0-.459-.458H9.71ZM3.75 14.75A2.753 2.753 0 0 1 1 12a2.753 2.753 0 0 1 2.75-2.75A2.753 2.753 0 0 1 6.5 12a2.753 2.753 0 0 1-2.75 2.75Zm0-4.583A1.835 1.835 0 0 0 1.917 12c0 1.011.822 1.833 1.833 1.833A1.835 1.835 0 0 0 5.583 12a1.835 1.835 0 0 0-1.833-1.833ZM14.292 23H2.375A1.377 1.377 0 0 1 1 21.625v-2.75c0-.758.617-1.375 1.375-1.375h11.917c.758 0 1.375.617 1.375 1.375v2.75c0 .758-.617 1.375-1.375 1.375ZM2.375 18.417a.458.458 0 0 0-.458.458v2.75c0 .253.205.458.458.458h11.917a.458.458 0 0 0 .458-.458v-2.75a.458.458 0 0 0-.458-.458H2.375ZM20.25 23a2.753 2.753 0 0 1-2.75-2.75 2.753 2.753 0 0 1 2.75-2.75A2.753 2.753 0 0 1 23 20.25 2.753 2.753 0 0 1 20.25 23Zm0-4.583a1.835 1.835 0 0 0-1.833 1.833c0 1.011.822 1.833 1.833 1.833a1.835 1.835 0 0 0 1.833-1.833 1.835 1.835 0 0 0-1.833-1.833Z" fill="#fff"/></svg>
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M21.625 6.5H2.375A1.377 1.377 0 0 1 1 5.125v-2.75C1 1.617 1.617 1 2.375 1h19.25C22.383 1 23 1.617 23 2.375v2.75c0 .758-.617 1.375-1.375 1.375ZM2.375 1.917a.458.458 0 0 0-.458.458v2.75c0 .253.205.458.458.458h19.25a.458.458 0 0 0 .458-.458v-2.75a.458.458 0 0 0-.458-.458H2.375ZM21.625 14.75H9.71a1.377 1.377 0 0 1-1.375-1.375v-2.75c0-.758.616-1.375 1.375-1.375h11.916c.758 0 1.375.617 1.375 1.375v2.75c0 .758-.617 1.375-1.375 1.375ZM9.71 10.167a.458.458 0 0 0-.459.458v2.75c0 .253.206.458.459.458h11.916a.458.458 0 0 0 .459-.458v-2.75a.458.458 0 0 0-.459-.458H9.71ZM3.75 14.75A2.753 2.753 0 0 1 1 12a2.753 2.753 0 0 1 2.75-2.75A2.753 2.753 0 0 1 6.5 12a2.753 2.753 0 0 1-2.75 2.75Zm0-4.583A1.835 1.835 0 0 0 1.917 12c0 1.011.822 1.833 1.833 1.833A1.835 1.835 0 0 0 5.583 12a1.835 1.835 0 0 0-1.833-1.833ZM14.292 23H2.375A1.377 1.377 0 0 1 1 21.625v-2.75c0-.758.617-1.375 1.375-1.375h11.917c.758 0 1.375.617 1.375 1.375v2.75c0 .758-.617 1.375-1.375 1.375ZM2.375 18.417a.458.458 0 0 0-.458.458v2.75c0 .253.205.458.458.458h11.917a.458.458 0 0 0 .458-.458v-2.75a.458.458 0 0 0-.458-.458H2.375ZM20.25 23a2.753 2.753 0 0 1-2.75-2.75 2.753 2.753 0 0 1 2.75-2.75A2.753 2.753 0 0 1 23 20.25 2.753 2.753 0 0 1 20.25 23Zm0-4.583a1.835 1.835 0 0 0-1.833 1.833c0 1.011.822 1.833 1.833 1.833a1.835 1.835 0 0 0 1.833-1.833 1.835 1.835 0 0 0-1.833-1.833Z"/></svg>
```

Comment:
It looks like it's better to remove `fill="none"` as well as we have no icon on preview in editors, which is not good